config/jobs/ppc64le-cloud/build-docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,166 @@
+periodics:
+    - name: periodic-config-docker
+      cluster: k8s-ppc64le-cluster
+      interval: 24h
+      decorate: true
+      decoration_config:
+        timeout: 4h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+          slack:
+            channel: 'prow-job-notifications'
+            report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:20890c8770ff57f91eb9f16fbb9f9e979958cd02b2a3c03795d2587e7b58a9db
```

Comment:
Why not use `latest` or the proper version tag here instead of SHA tag.
I see this SHA mapped to `0.7-ppc64le` tag at https://quay.io/repository/powercloud/docker-ce-build?tab=tags.

Reply:
We have made this move for all our jobs post a discussion where we concluded that hashes are immutable, hence more secure from intentional/accidental re-tagging.